### ISSUE TITLE
Deleting across answer does not delete down answer

### ISSUE THREAD

**Issue by jiggoha** [dejangvozdenac/crossword#11]

**Comment by dejangvozdenac** [comment]
idk i've been thinking about this and i kinda don't want this. like towards the end, if you wanted to delete a word on a completely filled board nothing would happen which is silly

**Comment by jiggoha** [comment]
Good point. What about thinking submitted answers as layers? And if you delete a word, it deletes that layer, but doesn't touch the other previously submitted acrosses or downs.

**Comment by dejangvozdenac** [comment]
As I said in the text I've literally never had a correct answer that I overwrote with an incorrect one
